extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -61,7 +61,7 @@ export const at = localize('at', "at");
 
 // commands
 export const revealFileInOsCommand = 'revealFileInOS';
-export const schemaCompareStartCommand = 'schemaCompare.start';
+export const schemaCompareStartCommand = 'SchemaCompare.start';
```

Comment:
Why did you change this? Commands typically start with a lower case letter (and I don't see the corresponding change in the package.json)

Reply:
This change will be reverted, accidentally updated the name and noticed after I sent to PR, have reverted already and waiting for more comments to push for new iteration.

Reply:
you don't have to worry about getting more comments before pushing out a new commit, even if it's just a small change. It's nice if the PR is updated after you make changes so the "old code" isn't reviewed :)